### PR TITLE
Add documentation generation

### PR DESCRIPTION
**Pull request by jaxxzer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+Not acknowledged.
+
+No payload
+
+### 3 ascii_text
+
+A message for transmitting text data.
+
+No payload
```
Comment (patrickelectric) [inline README.md]: This message has payload.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+Acknowledged.
+
+No payload
+
+### 2 nack
+
+Not acknowledged.
+
+No payload
```
Comment (patrickelectric) [inline README.md]: This message has payload

File: README.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+| u32 | ping_hz | The frequency of acoustic measurements. | Hz |
+| u32 | adc_sample_hz | The ADC sample frequency. | Hz |
+| u32 | ping_num | Ping count since boot. |  |
+| u32 | rms_goertzel_noise | RMS Goertzel noise. |  |
+
+### 1400 continuous_start
+
+Command to initiate continuous data stream of profile messages.
+
+No payload
```
Comment (patrickelectric) [inline README.md]: This and 1401 also has payload.

**Comment by jaxxzer** [comment]
It's no problem, I updated it and committed it in 30 seconds.

**Comment by jaxxzer** [comment]
We can add a bot/ci check :)

**Comment by patrickelectric** [comment]
I agree, the documentation should exist, we should add a CI test for that in the future. I can do that after this PR get merged. 

Have you pushed the updates ?

**Comment by jaxxzer** [comment]
Yes. I just hadn't run the generator again after my rebase in the first place.

**Pull request merged**